bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -176,13 +178,21 @@
 		<item-type>String</item-type>
 		<label>Play Mode</label>
 		<description>The Current Playback Mode</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Mode</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="timeElapsed">
 		<item-type>Number:Time</item-type>
 		<label>Playback Time</label>
 		<description>The Current Playback Time Elapsed</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Duration</tag>
```

Comment:
I'm in doubt here, probably because English is not my first language. I guess elapsed can be considered as "duration so far". But let's say you've watched the first 45 minutes of a 2 hour movie, probably you'd say that 45 minutes elapsed, but duration is 2 hours? I seriously don't know if this is correct and fine, or confusing, so hence asking.

Reply:
Hmm, yes I don't fully understand these either. "Duration so far" is what I was thinking as to what the 'Status' was indicating. Compared to the total duration which would not be a status that updates while the movie was playing.

Reply:
@andrewfg - can you help us out with your mother language here? 🙂

Reply:
The `Property.Duration` just says that what you see on screen has something to do with an interval of time.

However see my PR about UPNP where there is a discussion if media attributes Genre, Title, Album, Artist, Composer, .. Duration should be tagged with something like `Property.TrackInfo` ..

Reply:
> The `Property.Duration` just says that what you see on screen has something to do with an interval of time.

By that interpretation, I think what I have above is OK.